Celeste.Mod.mm/Patches/OuiFileSelectSlot.cs

```diff
@@ -338,6 +356,20 @@ class PatchOuiFileSelectSlotOnContinueSelectedAttribute : Attribute { }
 
     static partial class MonoModRules {
 
+        public static void PatchOuiFileSelectSlotUpdate(ILContext context, CustomAttribute attrib) {
+            TypeDefinition t_CoreModule = MonoModRule.Modder.Module.GetType("Celeste.Mod.Core.CoreModule");
+            TypeDefinition t_CoreModuleSettings = MonoModRule.Modder.Module.GetType("Celeste.Mod.Core.CoreModuleSettings");
+            MethodDefinition m_CoreModule_get_Settings = t_CoreModule.FindMethod("get_Settings");
+            MethodDefinition m_get_AllowScreenFlash = t_CoreModuleSettings.FindMethod("get_AllowScreenFlash");
+
+            ILCursor cursor = new ILCursor(context);
+            cursor.GotoNext(MoveType.Before, instr => instr.MatchLdsfld("Celeste.Settings", "Instance"));
+            cursor.RemoveRange(2);
+            cursor.EmitCall(m_CoreModule_get_Settings);
+            cursor.EmitCall(m_get_AllowScreenFlash);
```

Comment:
```suggestion
            cursor.EmitCallvirt(m_get_AllowScreenFlash);
```

Same here - `AllowScreenFlash` is an instance property.

Reply:
Addressed in commit [a87a04f](https://github.com/EverestAPI/Everest/pull/836/commits/a87a04fdafc78728fdb24bff92cbe487f25b805b)